charts/nucliadb_writer/templates/writer.vs.yaml

```diff
@@ -10,22 +10,68 @@ spec:
   http:
     - name: nucliadb_writer
       match:
+        # KBs (only for onprem)
         - uri:
             regex: '^/api/v\d+/kb/[^/]+'
           method:
             regex: "POST|PATCH|PUT|DELETE|OPTIONS"
-        - uri:
-            regex: '^/api/v\d+/kb/[^/]+/(resource|slug|resources|entitiesgroup|labelset|upload|custom-synonyms|export|import).*'
-          method:
-            regex: "POST|PATCH|PUT|DELETE|OPTIONS"
+
+        # KB config
         - uri:
             regex: '^/api/v\d+/kb/[^/]+/configuration'
           method:
             regex: "POST|PATCH|OPTIONS"
         - uri:
-            regex: '^/api/v\d+/kb/[^/]+/vectorsets/.*'
+            regex: '^/api/v\d+/kb/[^/]+/vectorsets/[^/]+$'
           method:
             regex: "POST|DELETE|OPTIONS"
+
+        # resources
+        - uri:
+            regex: '^/api/v\d+/kb/[^/]+/resources'
+          method:
+            regex: "POST|OPTIONS"
+        - uri:
+            regex: '^/api/v\d+/kb/[^/]+/(resource|slug)/[^/]+$'
+          method:
+            regex: "PATCH|DELETE|OPTIONS"
+
```

Comment:
Missing route?

`PATCH/DELETE/OPTIONS /(resource|slug)/{rid}`.

for
modify_resource_rslug_prefix
modify_resource_rid_prefix
delete_resource_rslug_prefix
delete_resource_rid_prefix

Reply:
me being very blind